lib/lp_token_auth/core.rb

```diff
@@ -56,5 +60,23 @@ def check_id!(id)
         raise LpTokenAuth::Error, "id must be a string or integer, you provided #{id}"
       end
     end
+
+    private
+
+    def private_key
+      # raise LpTokenAuth::NoKeyPresent if ENV['JWE_PRIVATE_KEY'].blank?
```

Comment:
@dgutt17 Can we remove this commented out code if the `begin.....rescue` is elsewhere?

Reply:
@bhennes2  I added some more specific error handling to the `private_key` method. Let me know what you think